src/languages/en.ts

```diff
@@ -853,6 +853,7 @@ const translations = {
         cameraErrorTitle: 'Camera error',
         cameraErrorMessage: 'An error occurred while taking a photo. Please try again.',
         locationAccessTitle: 'Allow location access',
+        locationRelaunchTitle: 'Relaunch now?',
```

Comment:
This doesn't seem to be used anywhere. Can we remove it?

Reply:
@luacmartins done